Extensions/ILSpy.Decompiler/dnSpy.Decompiler.ILSpy.Core/CSharp/CSharpDecompiler.cs

```diff
@@ -521,5 +521,16 @@ sealed class SelectFieldTransform : IAstTransform {
 				return DecompiledBodyKind.Full;
 			return DecompiledBodyKind.Empty;
 		}
+
```

Comment:
This file and many more now use LF line endings instead of CRLF line endings. Needs to be fixed.